HTML/HTML.sublime-syntax

```diff
@@ -227,6 +227,10 @@ contexts:
       captures:
         1: punctuation.definition.entity.html
         2: punctuation.terminator.entity.html
+    - match: (%)\h{2}
+      scope: constant.character.escape.html
```

Comment:
maybe better to be more specific here - `constant.character.escape.url.html`, in case users want to target it specially in their color schemes etc.?